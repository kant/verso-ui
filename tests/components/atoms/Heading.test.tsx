import { jsx } from '@emotion/react'
import renderer from 'react-test-renderer'
import { Heading } from 'src/components'

test('Snapshot Test: Heading Level One', () => {
  const component = renderer.create(
    <Heading element="h1">Heading Level One</Heading>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Two', () => {
  const component = renderer.create(
    <Heading element="h2">Heading Level One</Heading>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Three', () => {
  const component = renderer.create(
    <Heading element="h3">Heading Level One</Heading>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('Snapshot Test: Heading Level Four', () => {
  const component = renderer.create(
    <Heading element="h4">Heading Level One</Heading>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
