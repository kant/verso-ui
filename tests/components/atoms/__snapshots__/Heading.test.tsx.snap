// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Heading Level Four 1`] = `
<h4
  className="css-1eiceoa-Heading"
>
  Heading Level One
</h4>
`;

exports[`Snapshot Test: Heading Level One 1`] = `
<h1
  className="css-1eiceoa-Heading"
>
  Heading Level One
</h1>
`;

exports[`Snapshot Test: Heading Level Three 1`] = `
<h3
  className="css-1eiceoa-Heading"
>
  Heading Level One
</h3>
`;

exports[`Snapshot Test: Heading Level Two 1`] = `
<h2
  className="css-1eiceoa-Heading"
>
  Heading Level One
</h2>
`;
